docs/sources/tempo/metrics-generator/active-series.md

```diff
@@ -0,0 +1,118 @@
+| Service 1 | span1     | SPAN_KIND_INTERNAL | STATUS_CODE_ERROR | 6            |
+| Service 1 | span1     | SPAN_KIND_CLIENT   | STATUS_CODE_OK    | 23           |
+| Service 1 | span1     | SPAN_KIND_CLIENT   | STATUS_CODE_ERROR | 3            |
+
+Because of the variation in values, we now have four active series for our metric instead of one. But, as far as Service 1 is concerned, there's still only four active series, because there isn't any other variation of the values for labels. You can run 1 trace, 10 traces, 100 traces (each with however many loops of spans there are) and only four active series will ever be produced.
+
+In reality, we've only told half the story in our last diagram. We called a second service, _Service 2_, which continues the trace by adding a new span, `span2`.
+If there was a loop inside Service 2 with a single span that was generated from an upstream call from Service 1, and then a number of spans that were driven internally, which could also error, we'd end up with the possible values in the metric for `traces_spanmetrics_call_total` below:
+
+| service   | span_name | span_kind          | status_code       | Metric value |
+| --------- | --------- | ------------------ | ----------------- | ------------ |
+| Service 1 | span1     | SPAN_KIND_INTERNAL | STATUS_CODE_OK    | 89           |
+| Service 1 | span1     | SPAN_KIND_INTERNAL | STATUS_CODE_ERROR | 13           |
+| Service 1 | span1     | SPAN_KIND_CLIENT   | STATUS_CODE_OK    | 44           |
+| Service 1 | span1     | SPAN_KIND_CLIENT   | STATUS_CODE_ERROR | 9            |
+| Service 2 | span2     | SPAN_KIND_SERVER   | STATUS_CODE_OK    | 30           |
+| Service 2 | span2     | SPAN_KIND_SERVER   | STATUS_CODE_ERROR | 14           |
+| Service 2 | span2     | SPAN_KIND_INTERNAL | STATUS_CODE_OK    | 99           |
+| Service 2 | span2     | SPAN_KIND_INTERNAL | STATUS_CODE_ERROR | 23           |
+
+At this point, all our traces will be composed of two potential span names, each of which produce two separate types of `span_kind` and two separate types of `status_code`. So we have eight active series for a metric.
+
+The variability of values for each potential span condition determines the number of active series being produced by Tempo when ingesting spans for a trace, and not the number of traces of spans that are seen.
+
+## Custom span attributes
+
+Finally, there's one other consideration for active series: extra label key/value pairs that can be added onto metrics.
+The Tempo metrics generator allows the user to use arbitrary span attributes to be created as label pairs for metrics.
+When considering the number of active series generated, you also need to determine how many possible values there are for the span attribute being turned into a label.
+
+For example, if someone added an `http.method` span attribute into a metric label pair, there are five possible values (because there are five REST methods):
```

Comment:
```suggestion
For example, if you added an `http.method` span attribute into a metric label pair, there are five possible values (because there are five possible REST methods):
```